Mathlib/Algebra/BigOperators/Order.lean

```diff
@@ -417,6 +417,17 @@ theorem prod_mono_set' (f : ι → M) : Monotone fun s ↦ ∏ x in s, f x := fu
 #align finset.prod_mono_set' Finset.prod_mono_set'
 #align finset.sum_mono_set Finset.sum_mono_set
 
+@[to_additive]
+lemma apply_le_prod_of_mem (i : ι) (hi : i ∈ s) :
```

Comment:
Do you want both of those arguments to be explicit?

Reply:
Good point; I have made `i` implicit.